packages/squiggle-lang/src/rescript/Distributions/GenericDist/GenericDist.res

```diff
@@ -32,6 +32,15 @@ let normalize = (t: t): t =>
   | SampleSet(_) => t
   }
 
+let integralEndY = (t: t): float =>
```

Comment:
unconvinced `integralEndY` is the best name. It's the sum as _`x`_ goes from negative to positive infinity, unclear what `y` has to do with it.

Reply:
The integral is a cdf, which goes from 0 to 1. I agree it might not be the best name, but this is what it's already called in PointSetDist.T.